var/spack/repos/builtin/packages/hiop/package.py

```diff
@@ -19,7 +19,7 @@ class Hiop(CMakePackage, CudaPackage, ROCmPackage):
 
     homepage = "https://github.com/LLNL/hiop"
     git = "https://github.com/LLNL/hiop.git"
-    maintainers = ["ashermancinelli", "CameronRutherford", "pelesh"]
+    maintainers = ["ryandanehy", "CameronRutherford", "pelesh"]
```

Comment:
It's been a good run y'all 😭

Reply:
It has been! @ryandanehy can you also make the same change to the ExaGO package?